src/gui/QvisGUIApplication.C

```diff
@@ -2693,10 +2701,13 @@ QvisGUIApplication::SetOrientation(int orientation)
 //   Added logic specific to restoring session, to prevent window from
 //   being drawn partly offscreen when running VisIt remotely.
 // 
+//   Kathleen Biagas, Fri Apr 12 15:05:18 PDT 2019
+//   Remove orienation from args, it is now an ivar.
```

Comment:
You have a typo in the spelling of "orientation".

Reply:
Good catch on the typos, thanks. I fixed them online. Does that mean they will be fixed when I merge?